### ISSUE TITLE
Accessibility Audit: Static site Getting started section

### ISSUE THREAD

**Issue by mehansen** [CDCgov/prime-simplereport#5766]
### Background
The SimpleReport team performs quarterly accessibility audits as a proactive approach to ensure our products comply with Section 508 of the Rehabilitation Act.

### Action requested
Use Axe DevTools to perform an accessibility audit of the pages in the "Getting started" section of the static site. Do the page scan as well as any relevant guided tests.
Pages to visit:
- https://dev3.simplereport.gov/getting-started/
- https://dev3.simplereport.gov/getting-started/organizations-and-testing-facilities/onboard-your-organization/
- https://dev3.simplereport.gov/getting-started/organizations-and-testing-facilities/where-does-simplereport-work/
- https://dev3.simplereport.gov/getting-started/organizations-and-testing-facilities/test-devices-we-support/
- https://dev3.simplereport.gov/getting-started/organizations-and-testing-facilities/activate-your-user-account/
- https://dev3.simplereport.gov/getting-started/organizations-and-testing-facilities/log-in-to-simplereport/
- https://dev3.simplereport.gov/getting-started/organizations-and-testing-facilities/get-training/
- https://dev3.simplereport.gov/getting-started/public-health-departments/about-prime-reportstream/
- https://dev3.simplereport.gov/getting-started/public-health-departments/simplereport-data-catalog/

### Acceptance Criteria
Full audit completed and tickets created for any accessibility violations.

**Comment by johanna-skylight** [comment]
Tickets created:

https://github.com/CDCgov/prime-simplereport/issues/5809
https://github.com/CDCgov/prime-simplereport/issues/5808
https://github.com/CDCgov/prime-simplereport/issues/5806
https://github.com/CDCgov/prime-simplereport/issues/5805
